headless/server/src/main/java/com/tencent/supersonic/headless/server/rest/TagController.java

```diff
@@ -150,6 +150,7 @@ public ItemValueResp queryTagValue(@RequestBody ItemValueReq itemValueReq,
                                        HttpServletRequest request,
                                        HttpServletResponse response) throws Exception {
         User user = UserHolder.findUser(request, response);
+        itemValueReq.setDateConf(null);
```

Comment:
If it is set to null here, all requests can only use the same default date, but there may be cases where the default date of some tags is empty data, which is not suitable for the backend interface. It is recommended to detect the maximum date of the data, just as the logic of correctDateConf. If there are special similar requirements, it is recommended to add a new interface setting that meets the business scenario.

Reply:
Ok, I cancel the commit of the second point, is it useful for the first one to resolve the metric and dimension conflicts in the metric exploration?